app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,125 @@
+$opacity-light: rgba(0,0,0,0.2);
+$opacity-dark: rgba(0,0,0,0.5);
+$denied-color: rgb(200,0,0);
+$black: black;
+$white: white;
+
+.comments-wrapper {
+  width: 60%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 15px;
+  border-top: 1px solid $opacity-light;
+
+  .comments-container {
+    border-top: 1px solid $opacity-light;
+
+    .new-comment {
+      width: inherit;
+      margin-top: 30px;
+
+      h4 {
```

Comment:
Maybe it's too much work to change it now, but keep in mind for the future that nesting with more than 2 levels is not a very good practice.